x-pack/test/detection_engine_api_integration/security_and_spaces/tests/migrations.ts

```diff
@@ -97,94 +96,6 @@ export default ({ getService }: FtrProviderContext): void => {
           '7d'
         );
       });
-
-      it('migrates legacy siem-detection-engine-rule-status to use saved object references', async () => {
```

Comment:
Well this one had a nice long life 😅 -- happy to see this cleanup though! 🙂 

I hadn't thought about it till now, but there's no carry-over from the previous status object, so users upgrading will lose their last execution status, correct? I remember this being fine when we last discussed this, so no issue, just curious of the behavior on upgrade here.

Reply:
> Well this one had a nice long life 😅 -- happy to see this cleanup though! 🙂

Haha yeah. When I removed the SO, these tests broke. I'm curious to learn more about the underlying logic of FTR runner and these tests.

> users upgrading will lose their last execution status, correct? I remember this being fine when we last discussed this

Yup, correct